drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
Why is this piping to `sudo sh` and not just `sh`?  The script does intelligently `sudo` (or `su` as a fallback if `sudo` is unavailable, which isn't uncommon) itself only for the lines where `root` is necessary: https://github.com/docker/docker/blob/9f2c48614497dee2448b65c81aa6d44953484372/project/install.sh#L42-L53